### PR TITLE
[12.0][FIX] web_company_color: changed scss sanitized values

### LINKED ISSUE #1366: web_company_color init issue for first use
Please change a code for func: _scss_get_sanitized_values() like this. It will generate in right way

`    @api.multi
    def _scss_get_sanitized_values(self):
        self.ensure_one()
        values = dict(self.company_colors)
        values['color_navbar_text'] = self.color_navbar_text or '#FFF'
        values['color_navbar_bg'] = self.color_navbar_bg or '$o-brand-odoo'
        values['color_navbar_bg_hover'] = self.color_navbar_bg_hover or '$o-navbar-inverse-link-hover-bg'
        return values`

### PR DESCRIPTION
**Pull request by Tardo**
Fix #1366 and refactor the code a bit:
* Improved tests
* "Detailed" comments

### CODE REVIEW

File: web_company_color/models/res_company.py [hunk 1]
```diff
@@ -106,12 +106,16 @@ def write(self, values):
...
     def _scss_get_sanitized_values(self):
         self.ensure_one()
+        # Clone company_color as dictionary to avoid ORM operations
+        # This allow extend company_colors and only sanitize selected fields
+        # or add custom values
         values = dict(self.company_colors)
         values.update({
-            'color_navbar_bg': values['color_navbar_bg'] or '$o-brand-odoo',
-            'color_navbar_bg_hover': values['color_navbar_bg_hover']
+            'color_navbar_bg': values.get('color_navbar_bg')
```
Comment (pedrobaeza) [inline web_company_color/models/res_company.py]: Put dictionary value between parenthesis for avoiding wrong Python parsing.

**Comment by Tardo** [comment]
@pedrobaeza done

**Review by pedrobaeza — APPROVED** [review]

File: web_company_color/models/res_company.py [hunk 2]
```diff
@@ -106,12 +106,17 @@ def write(self, values):
     @api.multi
     def _scss_get_sanitized_values(self):
         self.ensure_one()
+        # Clone company_color as dictionary to avoid ORM operations
+        # This allow extend company_colors and only sanitize selected fields
+        # or add custom values
         values = dict(self.company_colors)
```
Comment (simahawk) [inline web_company_color/models/res_company.py]: this could be more safe like `dict(self.company_colors or {})`

**Review by simahawk — APPROVED** [review]

File: web_company_color/models/res_company.py (hunk 2 above)
Comment (Tardo) [reply web_company_color/models/res_company.py]: Done!

**Pull request merged**